.github/workflows/superset-python-integrationtest.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         mysql+mysqldb://superset:superset@127.0.0.1:13306/superset?charset=utf8mb4&binary_prefix=true
     services:
       mysql:
-        image: mysql:5.7
+        image: mysql:8.0
```

Comment:
Not sure if this was intended as part of this PR 🤷🏼‍♂️

Reply:
Yes I should have added a note, but my new unit test `test_virtual_dataset_with_comments` uses "in-a-subquery CTE", which apparently isn't supported in mysql `5.7`, failing that one test against old mysql. That CTE is one way that I was able to reproduce some of the issues from the previous reverted PR.
<img width="983" alt="Screenshot 2024-06-20 at 12 29 04 PM" src="https://github.com/apache/superset/assets/487433/4c3974dd-c4fd-4c3e-a670-e1db2bc0514b">